doc/advanced-configuration/outgoing-connections.md

```diff
@@ -188,278 +103,144 @@ sslmode     = verify-full
 
-## Riak connection setup
+#### `outgoing_pools.redis.*.connection.password`
+* **Syntax:** string
+* **Default:** `""`
+* **Example:** `password = "topsecret"`
 
-Currently only one Riak connection pool can exist for each supported XMPP host.
-It is configured with the following tuple inside the `outgoing_pools` config option.
+## Riak options
 
-```erlang
-{outgoing_pools, [
- {riak, global, default, [{workers, 20}], [{address, "127.0.0.1"}, {port, 8087}]}
-]}.
-```
+Currently only one Riak connection pool can exist for each supported XMPP host (the default pool).
 
 *WARNING:* `riak` backend is not compatible with `available_worker` strategy.
 
-#### Riak SSL connection setup
-
-Using SSL for Riak connection requires passing extra options in `ConnectionOptions` to the
-aforementioned `riak` tuple.
+#### `outgoing_pools.riak.*.connection.address`
+* **Syntax:** string
+* **Example:** `address = "127.0.0.1"`
 
-Here is the proper syntax:
+#### `outgoing_pools.riak.*.connection.port`
+* **Syntax:** integer
+* **Example:** `port = 8087`
 
-* **Credentials**
-    * **Description:** Specifies credentials to use to connect to the database.
-    * **Syntax:** `{credentials, User, Password}`
-    * **Supported values** `User` and `Password` are strings with a database username and password respectively.
+#### `outgoing_pools.riak.*.connection.credentials`
+* **Syntax:** `{user = "username", password = "pass}`
+* **Default:** none
+* **Example:** `credentials = {user = "myuser", password = "tisismepasswd"}`
+* **Comment:** optional - setting this option forces connection over TLS
 
-* **CACert**
-    * **Description:** Specifies a path to the CA certificate that was used to sign the database certificates.
-    * **Syntax:** `{cacertfile, Path}`
-    * **Supported values** `Path` is a string with a path to the CA certificate file.
+Riak also supports all TLS-specific options described in the TLS section.
 
-* **SSL_Opts**
-    * **Description**: list of SSL options as defined in [Erlang SSL module doc](http://erlang.org/doc/man/ssl.html)
-      They will be passed verbatim to the `ssl:connect` function.
-    * **Syntax** `{ssl_opts, ListOfSSLOpts}`
-    * **Example**:
+## Cassandra options
 
-```erlang
-{ssl_opts, [{ciphers, ["AES256-SHA", "DHE-RSA-AES128-SHA256"]},
-            {server_name_indication, disable}]}
-```
+#### `outgoing_pools.cassandra.*.connection.servers`
+* **Syntax:** a list of maps containing keys `"ip_adddress"` and `"port"`
+* **Default:** `[{ip_address = "localhost", port = 9042}]`
+* **Example:** `servers = [{ip_address = "host_one", port = 9042}, {ip_address = "host_two", port = 9042}]`
 
-##### Example configuration
+#### `outgoing_pools.cassandra.*.connection.keyspace`
+* **Syntax:** string
+* **Default:** `"mongooseim"`
+* **Example:** `keyspace = "big_mongooseim_database"`
 
-An example configuration can look as follows:
+To use plain text authentication (utilising cqerl_auth_plain_handler module):
```

Comment:
```suggestion
To use plain text authentication (using cqerl_auth_plain_handler module):
```
Or maybe
```suggestion
To use plain text authentication (making use of the cqerl_auth_plain_handler module):
```
But first option is better IMHO. In general "utilizing" is too bombastic and very very infrequently used in common English. It even entered the English dictionary too recently as a Latin loan for scientists, expected to mean pretty much the same than "using" except for tiny nuances, but nobody has ever agreed on what those nuances are anyway 😛

Reply:
I agree that "using" is simpler and more appropriate than "utilising". And "utilizing" is out of question, because it is American.